common/admin/telemetry/telemetry.go

```diff
@@ -0,0 +1,90 @@
+//go:build ceph_preview
+// +build ceph_preview
+
+package telemetry
+
+import (
+	"fmt"
+
+	ccom "github.com/ceph/go-ceph/common/commands"
+	"github.com/ceph/go-ceph/internal/commands"
+)
+
+// Admin is used to administer ceph nfs features.
+type Admin struct {
+	conn ccom.RadosCommander
+}
+
+// NewFromConn creates an new management object from a preexisting
+// rados connection. The existing connection can be rados.Conn or any
+// type implementing the RadosCommander interface.
+func NewFromConn(conn ccom.RadosCommander) *Admin {
+	return &Admin{conn}
+}
+
+// Status is the output of "ceph telemetry status"
+type Status struct {
+	ChannelBasic      bool   `json:"Channel_basic"`
+	ChannelCrash      bool   `json:"Channel_crash"`
+	ChannelDevice     bool   `json:"Channel_device"`
+	ChannelIdent      bool   `json:"Channel_ident"`
+	ChannelPerf       bool   `json:"Channel_perf"`
+	Contact           string `json:"Contact"`
+	Description       string `json:"Description"`
+	DeviceURL         string `json:"Device_url"`
+	Enabled           bool   `json:"Enabled"`
+	Interval          int    `json:"Interval"`
+	LastOptRevision   int    `json:"Last_opt_revision"`
+	Last_upload       string `json:"Last_upload"`
```

Comment:
Make check failing on this line. The name should be something like LastUpload to fit the Go standards